client/platform/desktop/constants.ts

```diff
@@ -213,6 +215,7 @@ export interface CheckMediaResults {
   websafe: boolean;
   originalFpsString: string;
   originalFps: number;
+  videoDimensions: {width: number; height: number};
```

Comment:
what's going on here?  Is this related to nist?

Reply:
Multiple activities have no spatial information and all tracks have associated bounding boxes.  This is used to organize the NIST activities on the screen in a row on the left side of the screen so you can see them visually.  Was going to do full frame but then overlapping activities would be unreadable in the viewer.